lib/gringotts/gateways/mercadopago.ex

```diff
@@ -0,0 +1,376 @@
+defmodule Gringotts.Gateways.Mercadopago do
+  @moduledoc """
+  [MERCADOPAGO][home] gateway implementation.
```

Comment:
Use the gateway branding style when you write docs, to my best knowledge they've named their product as `mercadopago`.